src/styles/blocks/card.scss

```diff
@@ -0,0 +1,168 @@
+@import '../utils/variables';
+
+.cards-container {
+  box-sizing: border-box;
+
+  --column-count: 1;
```

Comment:
Consider using CSS custom properties (variables) for colors and sizes throughout your styles to maintain consistency and make future changes easier.